sky/specs/style-guide.md

```diff
@@ -8,44 +7,114 @@ that everyone, whether reading the code for the first time or
 maintaining it for years, can quickly determine what the code does. A
 secondary goal is avoiding arguments when there are disagreements.
 
-All languages
--------------
+## All languages
 
-Avoid checking in commented-out code. It will bitrot too fast to be
-useful, and will confuse people maintaining the code.
+### AVOID checking in commented-out code
```

Comment:
The all-caps "AVOID" and "DON'T" and so on is pretty ugly. Can we do something more typographically pleasing?

In general having headings makes sense (the existing style guide is a wall of text with no headings organising it), but I'm not sure having the rules in the headings make sense. Why, for example, does "AVOID checking in comments that ask questions" have a heading, but "If commenting on a workaround due to a bug, also leave a link to the bug and a TODO to clean it up when the bug is fixed." not? Why is the heading "AVOID checking in comments that ask questions" rather than "DO find the answers to the questions, or describe the confusion, including references to where you found answers"?

Should the headings be more things like "Comments", "Line breaking" etc?